examples/z_put.py

```diff
@@ -70,7 +71,7 @@ def main():
     session = zenoh.open(conf)
 
     print("Putting Data ('{}': '{}')...".format(key, value))
-    session.put(key, value)
+    session.put(key, value, attachment={'key': b'value'})
```

Comment:
It would be better if we follow the same way in Zenoh example (Able to pass from argument)
https://github.com/eclipse-zenoh/zenoh/blob/0283aaae480d0c0608802a6fbfb79f7fc681469f/examples/examples/z_pub.rs#L63

Reply:
Indeed, I will fix that.